### PR TITLE
Add minimal startup code

### PR DESCRIPTION
**Pull request by heidebro**
Import and rename startup code.
Remove superfluous interrupt handlers and external initialization.
Initialize FPU manually.

### CODE REVIEW

File: third_party/custom_crt0/startup_stm32f407xx_stripped.s [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/**
+  ******************************************************************************
+  * @file      startup_stm32f407xx.s
```
Comment (marbre) [inline third_party/custom_crt0/startup_stm32f407xx_stripped.s:3]:
```suggestion
 * @file      startup_stm32f407xx_stripped.s
```

A general remark, should we rename the file to `startup_stm32f4xx.s`? No real need to name it strict but also no need to name it 407. Maybe 4xx is the better choice?

**Review by marbre — CHANGES_REQUESTED** [review]

File: third_party/custom_crt0/startup_stm32f407xx_stripped.s (hunk 1 above)
Comment (heidebro) [reply third_party/custom_crt0/startup_stm32f407xx_stripped.s:3]: Sounds reasonable. I'm assuming the reference in the README to the `stm32f407` should be turned into `stm32f4xx` aswell then?

Comment (marbre) [reply third_party/custom_crt0/startup_stm32f407xx_stripped.s:3]: If you rename so `stm32f4xx`, please also change it there accordingly.

**Review by marbre — APPROVED** [review]
I would have dropped the `stripped` from the file too, but that is something we can also address in a follow up (if we still want to). Thanks for the work.

**Pull request merged**